pmm-app/src/pmm-dbaas/components/DBCluster/EditDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.constants.ts

```diff
@@ -0,0 +1,38 @@
+import { SelectableValue } from '@grafana/data';
```

Comment:
This file doesn't need to be duplicated. Use the one in the AddDBClusterModal folder instead.

Reply:
It's a philosophical question, I would say that in this case, it's better to violate DRY than create a too tight coupling between AddDBClusterModal and EditDBClusterModal. These components a little different, and maybe the difference will be bigger in the future so it's dangerous to use dependencies directly. What do you think?

Reply:
Totally agree with you. That's why we should keep the duplication in things like component itself. But constants and types? Do we really need the duplication? I doubt that we will have different topologies and resources in both modals for example. It's more a subjective question rather than philosophical 😀 So yeah it can go both sides, just pick one. Either one is fine to me, was just raising this concern.

Reply:
Decided to discuss it on a next sync call